app/controllers/authentication_controller.rb

```diff
@@ -37,6 +38,24 @@ def callback
     head :unauthorized
   end
 
+  def end_session
+    end_session_uri =
+      if using_digital_identity?
+        id_token = get_id_token_from_session
+        uri = oidc_client_class.new.end_session_endpoint
+        if id_token
+          querystring = Rack::Utils.build_nested_query(id_token_hint: id_token)
+          "#{uri}?#{querystring}"
+        else
+          uri
+        end
+      else
+        "#{Plek.find('account-manager')}/sign-out?continue=1"
+      end
```

Comment:
Maybe pull some/all of this logic out into a separate private method? 
```suggestion
      if using_digital_identity?
        digital_identity_end_session_url
      else
        "#{Plek.find('account-manager')}/sign-out?continue=1"
      end
```